websites/A/Anizium/metadata.json

```
@@ -0,0 +1,23 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.7",
+	"author": {
+		"id": "708746139092123679",
+		"name": "Mert^^#0123"
+	},
+	"service": "Anizium",
+	"description": {
+		"en": "Anizium is a website where you can watch anime with 4K Turkish subtitles and dubs."
+	},
+	"url": "anizium.com",
```

Site supports `www` subdomain
```suggestion
	"url": [
		"anizium.com",
		"www.anizium.com"
	],
```